repos/dir.targets

```diff
@@ -5,7 +5,7 @@
     <EnvironmentVariables Include="OfficialBuildId=$(OfficialBuildId)" />
     <EnvironmentVariables Include="VersionSeedDate=$(OfficialBuildId)" />
     <!-- Work around a dependency on .git that doesn't have a good fallback yet: https://github.com/dotnet/buildtools/issues/1980 -->
-    <EnvironmentVariables Include="GitBranchName=N/A" />
+    <EnvironmentVariables Include="RawGitBranchName=N/A" />
```

Comment:
Can remove this workaround completely, as discussed in https://github.com/dotnet/buildtools/issues/1980.